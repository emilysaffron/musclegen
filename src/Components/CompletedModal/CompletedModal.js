import styled from "@emotion/styled";
import SaveButton from "../SaveButton";
const StyledModal = styled.div`
  width: 450px;
  height: 450px;
  background-color: rgba(0, 0, 0, 0.9);
  color: white;
  position: fixed;
  z-index: 100;
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
`;

const CompletedModal = ({ modal, toggleModal, currentPlan }) => {
  return modal ? (
    <StyledModal>
      <SaveButton
        modal={modal}
        toggleModal={toggleModal}
        currentPlan={currentPlan}
      />
    </StyledModal>
  ) : null;
};
export default CompletedModal;
