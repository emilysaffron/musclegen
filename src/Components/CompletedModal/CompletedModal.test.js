import renderer from "react-test-renderer";
import CompletedModal from "./CompletedModal";

it("CompletedModal Snapshot", () => {
  const tree = renderer
    .create(
      <CompletedModal modal={true} toggleModal="toggleModal" currentPlan={[]} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
