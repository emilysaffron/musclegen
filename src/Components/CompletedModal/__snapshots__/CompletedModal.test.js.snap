// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompletedModal Snapshot 1`] = `
<div
  className="css-1uxov8o"
>
  <a
    aria-current="page"
    href="/"
    onClick={[Function]}
  >
    <div
      className="css-ov1ktg"
    >
      <button
        className="css-ljtnwa"
        onClick={[Function]}
      >
        Save Workout
      </button>
      <button
        className="css-ljtnwa"
        onClick={[Function]}
      >
        Don't Save
      </button>
    </div>
  </a>
</div>
`;
